src/ui/components/pages/Account/Account.tsx

```diff
@@ -31,7 +33,18 @@ export function Account(props: Props) {
 
     const { t } = useTranslation({ Account });
 
-    const tabs = useMemo(() => accountTabIds.map(id => ({ id, "title": t(id) })), [t]);
+    const availableDeploymentRegions = useSelector(
+        state => state.deploymentRegion.availableDeploymentRegions,
+    );
+    const kubernetesServerUrl = availableDeploymentRegions[0].kubernetes?.url || "";
```

Comment:
- This is not the kind of logic that you do on the UI.  
- You can't assume it's the first region that is currently in use.  (What if, on AWS we are on `eu-frankfurt` and you provide the credential for a cluster in `us-arisona` just because you it was first in the list.  

For retreiving the selected region you would do:  

```tsx
import { selectors, useSelector } from "ui/coreApi";

function Component(){

    const { selectedDeploymentRegion } = useSelector(selectors.deploymentRegion.selectedDeploymentRegion);

    return <>...</>;
}
```

More info:  
https://docs.onyxia.sh/contributing/onyxia-web/architecture  
https://github.com/garronej/redux-clean-architecture

Reply:
@smith-julien This needs to be addressed.